### PR TITLE
previewctl also install gce contexts

### PR DESCRIPTION
**Pull request by vulkoingim**
## Description

* Install the context of gce VMs in the same manner as we do for harvester
* Remove the wait for VM ready - since we retry until the install succeeds, we're bound to retry if the vm/proxy is not ready, so we simplify things a bit.

## Related Issue(s)

Fixes #

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

**Comment by roboquat** [comment]
@vulkoingim: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: dev/preview/infrastructure/gce/proxy.tf [hunk 1]
```diff
@@ -16,10 +16,10 @@ resource "kubernetes_pod" "proxy" {
       command           = ["/bin/ash"]
       # dumb port-forward directly to the machine
       # those should be all the ports that need to be accessible in a preview env
-      # we also forward 22 to 2222, as 22 is disabled in the vm image and we don't want to change that
+      # we also forward 22,2200 to 2222, as 22 is disabled in the vm image and we don't want to change that
       args = [
         "-c",
-        "socat TCP-LISTEN:22,fork,reuseaddr TCP:${google_compute_instance.default.network_interface.0.access_config.0.nat_ip}:2222 & for i in 2200 80 443 6443 9090 3000; do socat TCP-LISTEN:$i,fork,reuseaddr TCP:${google_compute_instance.default.network_interface.0.access_config.0.nat_ip}:$i & done; wait"
+        "for i in 22 2200; do socat TCP-LISTEN:$i,fork,reuseaddr TCP:${google_compute_instance.default.network_interface.0.access_config.0.nat_ip}:2222 & done; for i in 80 443 6443 9090 3000; do socat TCP-LISTEN:$i,fork,reuseaddr TCP:${google_compute_instance.default.network_interface.0.access_config.0.nat_ip}:$i & done; wait"
```
Comment (meysholdt) [inline dev/preview/infrastructure/gce/proxy.tf:22]: Doe this code listen start a process for every port from 22 to 2200? This sound expensive, do we need that? Does this conflict the the code that tries to listen on sockes in the loop a few lines down? It looks like the ports overlap and on every port there can only one process listening at a given point in time.

Comment (vulkoingim) [reply dev/preview/infrastructure/gce/proxy.tf:22]: No, it's a list. In the first loop we forward 22, 2200 "local", to 2222 remote (see the comment). And the other one, forwards those other ports to the same remote ones.

Comment (meysholdt) [reply dev/preview/infrastructure/gce/proxy.tf:22]: 👍

**Comment by meysholdt** [comment]
Nit: Give `install-k3s.sh` a better name, e.g. `bootstrap-k3s.sh`, because inside the VM there is already another `/usr/local/bin/install-k3s.sh` and it will be confusing to have two scripts with the same name but different purposes. 

I see that in Harvester VMs, the script is already being stored under the name `bootsrap-k3s.sh`. It would be nice if the name would be the same in the VM and in the git-repo.
https://github.com/gitpod-io/gitpod/blob/0bd0957ef3aa6353955877ea74f31e924a15afa7/dev/preview/infrastructure/modules/harvester/cloudinit.yaml#L46-L50

**Comment by vulkoingim** [comment]
> Nit: Give `install-k3s.sh` a better name, e.g. `bootstrap-k3s.sh`, because inside the VM there is already another `/usr/local/bin/install-k3s.sh` and it will be confusing to have two scripts with the same name but different purposes.
> 
> I see that in Harvester VMs, the script is already being stored under the name `bootsrap-k3s.sh`. It would be nice if the name would be the same in the VM and in the git-repo.
> 
> https://github.com/gitpod-io/gitpod/blob/0bd0957ef3aa6353955877ea74f31e924a15afa7/dev/preview/infrastructure/modules/harvester/cloudinit.yaml#L46-L50

wrong pr

**Review by meysholdt — APPROVED** [review]
(also wrong PR :(  I'll take a look now and let you know if I find anything)

**Pull request merged**